api/views.py

```diff
@@ -375,6 +375,16 @@ def create(self, request, **kwargs):
 
             puzzle = serializer.save(hunt=hunt, chat_room=chat_room)
 
+            if (
+                "assigned_meta" in request.data
+                and request.data["assigned_meta"]
+                and request.data["assigned_meta"] != "none"
+            ):
+                meta = get_object_or_404(
+                    Puzzle, name=request.data["assigned_meta"], hunt=puzzle.hunt
+                )
+                puzzle.metas.add(meta)
```

Comment:
Should we add a unit test for this? (Can be in a separate PR if we want to expedite this one)

Reply:
Sure. I'll add a unit test.